doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
if we go the title route, can that please be documented somewhere? (Possibly in this doc). I know some of it is borrowed from [scipy](http://docs.scipy.org/doc/numpy-dev/dev/gitwash/development_workflow.html) but even a link to that would be helpful.